### DISCUSSION TITLE
AWS Secrets Manger

### DISCUSSION THREAD

**Discussion by mlanders** [hasura/graphql-engine#9195] [Question]
Is there a way to connect the Hasura cloud environment variables to AWS Secrets Manager. Would like to keep an API Key up to date even if the secret is rotated.

**Comment by rsd1122** [comment]
@mlanders - would love to understand more. Could you please send me an email at: rishi.divate@hasura.io and I'll get you in touch with the right folks at Hasura to take this forward.
